jdaviz/configs/imviz/plugins/orientation/orientation.vue

```diff
@@ -23,12 +23,20 @@
         </v-alert>
 
         <v-alert
-          v-if="wcs_linking_available"
+          v-if="wcs_linking_available && !need_clear_astrowidget_markers &&!need_clear_subsets"
           type='warning'
           class="ignore-api-hints"
           style="margin-left: -12px; margin-right: -12px"
         >
-          Switching link type will reset zoom.
+          Switching alignment will reset zoom.
+        </v-alert>
+
+        <v-alert
+          v-if="plugin_markers_exist && !need_clear_astrowidget_markers &&!need_clear_subsets"
+          type='warning'
+          style="margin-left: -12px; margin-right: -12px"
+        >
+          Marker positions may not be pixel-perfect when changing alignment/linking options.

```

Comment:
Why not?

Reply:
this was there before, I think from concerns by @pllim

Reply:
(the diff here is just from moving its position so that the "warnings" both come before the alerts with actions)

Reply:
I forgot the exact reason now but hopefully that concern was captured somewhere in this GitHub repo. Maybe related to this?

https://github.com/spacetelescope/jdaviz/blob/5130cafd8f8790a3e09b2d82c5ee721a1b909d9f/jdaviz/configs/imviz/tests/test_orientation.py#L74-L81

Reply:
ah, I think you're right.  That is still open, so let's leave the text in place and I'll make sure there is a link back here so that we remove the warning if/when addressing that.  Thanks!